lib/teslamate/vehicles/vehicle/summary.ex

```diff
@@ -13,7 +13,7 @@ defmodule TeslaMate.Vehicles.Vehicle.Summary do
     odometer shift_state charge_port_door_open time_to_full_charge charger_phases
     charger_actual_current charger_voltage version update_available update_version is_user_present geofence
     model trim_badging exterior_color wheel_type spoiler_type trunk_open frunk_open elevation power
-    charge_current_request charge_current_request_max
+    charge_current_request charge_current_request_max tpms_pressure_fl tpms_pressure_fr tpms_pressure_rl tpms_pressure_rr
```

Comment:
Fields added here are automatically published via MQTT. Could you please add them to the documentation: `website/docs/integrations/mqtt.md`

Reply:
Done :)